draft-ietf-quic-transport.md

```diff
@@ -317,23 +317,23 @@ lower-numbered streams of the same type in the same direction.
 
 QUIC allows for an arbitrary number of streams to operate concurrently.  An
 endpoint limits the number of concurrently active incoming streams by limiting
-the maximum stream ID (see {{stream-limit-increment}}).
+the number of streams (see {{stream-limit-increment}}).
 
-The maximum stream ID is specific to each endpoint and applies only to the peer
-that receives the setting. That is, clients specify the maximum stream ID the
-server can initiate, and servers specify the maximum stream ID the client can
+The stream limit is specific to each endpoint and applies only to the peer that
+receives the setting. That is, the client limits the number of streams the
+server can initiate, and the server limits the number of streams the client can
 initiate.  Each endpoint may respond on streams initiated by the other peer,
 regardless of whether it is permitted to initiate new streams.
 
 Endpoints MUST NOT exceed the limit set by their peer.  An endpoint that
 receives a STREAM frame with an ID greater than the limit it has sent MUST treat
-this as a stream error of type STREAM_ID_ERROR ({{error-handling}}), unless this
-is a result of a change in the initial limits (see {{zerortt-parameters}}).
+this as a stream error of type STREAM_LIMIT_ERROR ({{error-handling}}), unless
+this is a result of a change in the initial limits (see {{zerortt-parameters}}).
```

Comment:
Existing text, but....  We don't allow reductions in TPs after accepting 0-RTT, so how would it be the result of a change in the initial limits?

Reply:
Cutting that.  I didn't think much on it before, but you are right that it's nonsensical.